recipe/build-core.sh

```diff
@@ -1,8 +1,13 @@
-# First, build go1.4 using gcc, then use that go to build go>1.4
-export GOROOT_BOOTSTRAP=$SRC_DIR/go-bootstrap
-pushd $GOROOT_BOOTSTRAP/src
-./make.bash
-popd
+set -euf 
+
+#
+# Install and source the [de]activate scripts.
+for F in activate deactivate; do
+  mkdir -p "${PREFIX}/etc/conda/${F}.d"
+  cp -v "${RECIPE_DIR}/${F}-go-${cgo_var}.sh" "${PREFIX}/etc/conda/${F}.d/${F}-z60-go-${cgo_var}.sh"
+done
+
+source "${PREFIX}/etc/conda/activate.d/activate-z60-go-${cgo_var}.sh"
```

Comment:
Eating our own dogfood. This is mostly relevant to the `cgo` variant because it is the one that sets CGO_ENABLED, CGO_{C,CPP,CXX,F,LD}FLAGS and we want those to match what is coming from the gcc/clang/etc compilers.